### PR TITLE
change tsfile_manage_strategy default LEVEL_STRATEGY

### PR DESCRIPTION
**Pull request by zhanglingzhe0820**

### CODE REVIEW

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 1]
```diff
@@ -335,7 +335,7 @@ chunk_merge_point_threshold=20480
 
 # How many thread will be set up to perform hot compaction, 30 by default.
```
Comment (qiaojialin) [inline server/src/assembly/resources/conf/iotdb-engine.properties:336]:
```suggestion
# How many thread will be set up to perform hot compaction, 10 by default.
```

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java [hunk 2]
```diff
@@ -559,7 +559,7 @@
    * How many thread will be set up to perform hot compaction, 30 by default. Set to 1 when less
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java:559]:
```suggestion
   * How many thread will be set up to perform hot compaction, 10 by default. Set to 1 when less
```

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 1 above)
Comment (zhanglingzhe0820) [reply server/src/assembly/resources/conf/iotdb-engine.properties:336]: okay, I will solve itv

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java (hunk 2 above)
Comment (zhanglingzhe0820) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java:559]: okay, I will solve it

**Comment by qiaojialin** [comment]
duplicated with https://github.com/apache/iotdb/pull/1930

**Pull request closed without merging**